db/api.sql

```diff
@@ -1,5 +1,6 @@
 create table acos (
   uuid uuid not null primary key,
+  cms_id char(5) unique null,
```

Comment:
why is cms_id nullable?  for backwards compatibility?

Reply:
Yep, exactly. Without this we'd need to add lots of IDs matching the CMS format, and there aren't many available for testing purposes.